src/App.tsx

```diff
@@ -1,24 +1,217 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completedTodo,
+  updateTodoTitle,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterBy } from './types/FilterBy';
+import { Notification } from './components/Notification';
+import { NewTodo } from './types/NewTodo';
+import { ErrorNotifications } from './types/ErrorNotifications';
 
-const USER_ID = 0;
+const USER_ID = 6259;
+let isToggled = false;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [isError, setIsError] = useState(false);
+  const [errorText, setErrorText] = useState(ErrorNotifications.NONE);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [disableInput, setDisableInput] = useState(false);
+  const [editingTodosId, setEditingTodosId] = useState<number[]>([]);
+  const [formShowedForId, setFormShowedForId] = useState(0);
+
+  const getAllTodos = async () => {
+    try {
+      setDisableInput(true);
+      const data = await getTodos(USER_ID);
+
+      setTodos(data);
+      setDisableInput(false);
+    } catch (error) {
+      setErrorText(ErrorNotifications.LOADING);
+      setIsError(true);
+    }
+  };
+
+  useEffect(() => {
+    getAllTodos();
+  }, []);
+
+  const handleAddTodo = async (todoText: string) => {
+    if (!todoTitle.trim()) {
+      setIsError(true);
+      setErrorText(ErrorNotifications.TITLE);
+    } else {
+      const newTodoBody: NewTodo = {
+        title: todoText,
+        userId: USER_ID,
+        completed: false,
+      };
+
+      try {
+        setTodoTitle('');
+        await addTodo(USER_ID, newTodoBody);
```

Comment:
check in description of the task for adding todos, how you need to process it
you didn't complete all requirements
https://github.com/mate-academy/react_todo-app-add-and-delete